rero_ils/modules/acq_receipts/api.py

```diff
@@ -108,9 +111,41 @@ def _build_additional_refs(cls, data):
                 '$ref': get_ref_for_pid('org', order.organisation_pid)
             }
 
+    def create_receipt_lines(self, receipt_lines=None, dbcommit=True,
+                             reindex=True):
+        """Create multiple receipt lines.
+
+        :param receipt_lines: a list of dicts to create the records.
+        :param dbcommit: if True call dbcommit, make the change effective
+                         in db.
+        :param redindex: reindex the record.
+        :returns: a list containing the given data to build the receipt line
+                  with a `status` field, either `success` or `failure`.
+                  In case of `success`, the created record is returned.
+                  In case `failure`, the reason is given in a field `error`
+        """
+        created_receipt_lines = []
+        for receipt_line in receipt_lines:
```

Comment:
this will failed if receipt_lines is `None` as describe in arguments.
```
receipt_lines = receipt_lines or []
for receipt_line in receipt_lines:
    ...
```